tests/ChannelStatusTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+namespace tests;
+use Ably\AblyRest;
+
+require_once __DIR__ . '/factories/TestApp.php';
+
+
+class ChannelStatusTest extends \PHPUnit\Framework\TestCase
+{
+    protected static $testApp;
+    protected static $defaultOptions;
+    protected static $ably;
+
+    public static function setUpBeforeClass(): void
+    {
+        self::$testApp = new TestApp();
+        self::$defaultOptions = self::$testApp->getOptions();
+        self::$ably = new AblyRest(array_merge(self::$defaultOptions, [
+            'key' => self::$testApp->getAppKeyDefault()->string,
+        ]));
+    }
+
+    public static function tearDownAfterClass(): void
+    {
+        self::$testApp->release();
+    }
+
+    /**
+     * @testdox RSL8, CHD1
+     */
+    public function testChannelStatusAfterPublish()
+    {
+        $channel = self::$ably->channel('channel1');
+        $response = $channel->publish("event", "Hi there"); // publish the message
```

Comment:
From https://ably.com/docs/rest/channel-status#metadata-rest
> A side-effect of this request, in the current version of this API, is that it will cause the channel in question to become activated

So we don't actually need to publish anything, the channel status will always be active once we've made the request.

```suggestion
    public function testChannelStatus()
    {
        $channel = self::$ably->channel('channel1');
```

Reply:
Fixed